### ISSUE TITLE
Intelligent mockups

### ISSUE THREAD

**Issue by sirex** [sorl/django-mockups#10]
Now django-mockups is not very smart it just generates random and very log lorem ipsum lines. What do you think if generated values would be selected using some intelligent system?

Intelligent system should have a database of most common field names and rules how to populate these fields with realistic values.

One example would be first/last names. If field is named as 'first_name', then a name from names list should be selected, instead of log lorem ipsum line.

Before doing any implementations, I would like to hear other opinions about this feature...

**Comment by sorl** [comment]
Yes I have been wanting to do that since i started this actually, it would be good if there were localized paths to text files with csv content, ie en/first_name.txt, sv/content.txt. Just for convenience there could be a mapping from a common model field name to the textfile, that was my take on the matter. The reason for csv files (one entry per row) was that it is so easy for someone to update that and commit patch.
